### DISCUSSION TITLE
Can anyone give some examples of app built with flutter in Nepal ?

### DISCUSSION THREAD

**Discussion by bjuadk** [community/community#28037] [Other Feature Feedback, Questions, & Ideas] [answered]
I just wonder if there someone can help me with listing name of apps built with Flutter and used in Nepal

**Comment by sthakrishnakumar** [comment, marked as answer]
Dear @bjuadk ,
                       You raised a good question. You can find the lists below:
-Admission Nepal
-Nepal School Mela
-Virtual EduFair
-Khalti
-Ugbazar
-Nagarik App
-ePassport
-Moru
-WePlay
-Karobar
-myWorldlink app
-meroschool
-Merolagani nepse app
-Gorkha House
-DealKhana
-Ktm Food

**Reply by bjuadk** [reply]
Thank you so much for your effort
